src/wfs/src/test/java/org/geoserver/wfs/v2_0/GetCapabilitiesTest.java

```diff
@@ -777,4 +778,24 @@ public void testInternationalContentMultipleLanguages() throws Exception {
         assertXpathEvaluatesTo("resumé", fifteenLayer + "/wfs:Abstract", doc);
         assertXpathEvaluatesTo("parola chiave", fifteenLayer + "/ows:Keywords/ows:Keyword", doc);
     }
+
+    @Test
+    public void testAcceptLanguagesParameter() throws Exception {
+        Catalog catalog = getCatalog();
```

Comment:
Same as above.

Reply:
Catalog and FeatureTypeInfo are needed to have language links to appear.